src/server/components/annual-review/controller.ts

```diff
@@ -53,7 +76,7 @@ export async function confirmGetController(req: Request, res: Response, next: Ne
 
     res.render("annual-review/provider-confirmation", {
       rows,
-      country: listItem?.jsonData?.country,
+      country: listItem.address.country.name,
```

Comment:
I would actually change this to `listItem.list.country.name` as the address country is a separate selectable field in the application form and could potentially differ to the List country.